### ISSUE TITLE
[Card Updates] Latest Cards for Koishipro

### ISSUE THREAD

**Issue by dembercorrales01** [purerosefallen/ygopro#58]
Need BODE, KICO, SR12 and other new cards in Koishipro

**Comment by dembercorrales01** [comment]
lol this game sucks because of Master Duel

**Comment by yoyoys** [comment]
This game is okay because master duel has no Splight.
![df38x7k-6794f6b3-f07f-45c8-bd2f-a265c4cd3331](https://user-images.githubusercontent.com/22124891/168670254-694da2de-4aaa-4275-8007-085d590f9728.png)

**Comment by dembercorrales01** [comment]
yeah i know but also Koishipro have latest updates like Edopro? and currently released Duelist of Pyroxene?

**Comment by yoyoys** [comment]
> 
I think they share same library, just choose which client you like.

**Comment by dembercorrales01** [comment]
but why supporting this simulator that has latest updates and banlist
